src/doc/man/cargo-remove.md

```diff
@@ -0,0 +1,92 @@
+# cargo-remove(1)
+{{*set actionverb="Remove"}}
+{{*set nouns="removes"}}
+
+## NAME
+
+cargo-remove - Remove dependencies from a Cargo.toml manifest file
+
+## SYNOPSIS
+
+`cargo remove` [_options_] _dependency_...
+
+## DESCRIPTION
+
+Remove one or more dependencies from a `Cargo.toml` manifest.
+
+## OPTIONS
+
+### Section options
+
+{{#options}}
+
+{{#option "`--dev`" }}
+Remove as a [development dependency](../reference/specifying-dependencies.html#development-dependencies).
+{{/option}}
+
+{{#option "`--build`" }}
+Remove as a [build dependency](../reference/specifying-dependencies.html#build-dependencies).
+{{/option}}
+
+{{#option "`--target` _target_" }}
+Remove as a dependency to the [given target platform](../reference/specifying-dependencies.html#platform-specific-dependencies).
+{{/option}}
+
+{{/options}}
+
+### Miscellaneous Options
+
+{{#options}}
+
+{{#option "`--dry-run`" }}
+Don't actually write to the manifest.
+{{/option}}
+
+{{/options}}
+
+### Display Options
+
+{{#options}}
+{{> options-display }}
+{{/options}}
+
+### Manifest Options
+
+{{#options}}
+{{> options-manifest-path }}
+
+{{> options-locked }}
```

Comment:
Although `cargo rm` technically accepts `--locked/--frozen`, I don't think it should be documented here because it prevents `cargo rm` from working. It is somewhat a bug that `--locked/--frozen` are global flags. They should have been added on a per-command basis, but that may be difficult to fix now.

This may require separating `--offline` from `options-locked.md` so that it can be kept. (See also #11237)

Reply:
I can certainly move `--offline` from `options-locked.md` to resolve that. Is that something you'd like in a separate PR?

Ideally I imagine we'd make that change at the same time we make `--locked`/`--frozen` not global, so that the problem is fixed and documentation is correctly maintained, but maybe that's not a problem worth solving right now.

Reply:
side note: I think whatever we do, the shell completions should match the docs (see #11204)

Reply:
> Is that something you'd like in a separate PR?

Either here or separate is fine with me. Maybe separate might be better if it involves changing a lot of files?